### DISCUSSION TITLE
Way to export just performers

### DISCUSSION THREAD

**Discussion by stevensonjimmy** [stashapp/stash#4261] [General]
I currently use Stash on my PC. I'm attempting to install Stash on my Synology NAS using Docker. I've tried doing a full export from my current Stash but it keeps crashing before completing. I really just need to get all the Performers out because I can rebuild the rest. Is there any way to do this?

**Comment by DogmaDragon** [comment]
You can go to the Performers tab > click `...` > `Export all...`.

Also make a backup, most of the information you get from the export is stored in the database tables.

**Reply by stevensonjimmy** [reply]
Thank you! Had no idea that option was there.
